lib/resolvers.js

```diff
@@ -29,3 +30,17 @@ export const mapLinkList = fieldName => doc => {
     return linkItem;
   }).filter(l => l !== undefined);
 };
+
+export const mapScheduleList = fieldName => doc => {
+  const schedule = doc[fieldName];
+
+  if (!schedule) { return []; }
+  return schedule.map((item) => {
```

Comment:
```
export const mapScheduleList = fieldName => doc => {
  const scheduleList = doc[fieldName] || [];

  return scheduleList.reduce((list, schedule) => {
    const datetime = pathOr('', ['datetime', 'value'], item);
    const text = pathOr('', ['text', 'value'], item);

    return (datetime && text) 
      ? list.concat({ datetime, text })
      : list;
  }, []);
};
```

The return condition of the reducer's callback ensures we just return the list as-is if the `datetime` or `text` fields are falsey.